### PR TITLE
Coalesce 2022 Recap Blog

### PR DESCRIPTION
**Pull request by pkrish123**

### CODE REVIEW

File: posts/coalesce2022-a-recap.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+- Data
+- dbt
+- Python
+- Snowflake
+date: 2022-10-31T13:33:00.000Z
+title: "Coalesce 2022 - The Analytics Engineering Conference hosted by dbt Labs (A Recap)"
+image: 
+---
+
+Coalesce 2022 is dbt Labs' analytics engineering conference. For its third year, from October 17th to 21st, 2022, Coalesce was a whirlwind data extravaganza boasting speaker sessions and workshops in In-Person in London, Sydney, and New Orleans and virtually. Over 2500 people from all over the world attended in person, and over 20,000 individuals online. I had the pleasure of attending in-person in New Orleans over the week, and throughout this blog, I'll be presenting the Top Announcements of the week from dbt and some of my favorite sessions I attended while there.
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:14]: The "boasting speaker sessions and workshops in In-Person in London" sounds a bit strange... is the name "In-Person" in which case this grammatically makes sense but still _feels_ weird.

File: posts/coalesce2022-a-recap.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+The final product announcement in the keynote was of the dbt Semantic Layer. dbt prides itself on streamlining governance and managing the complexity of analytics data pipelines and business logic. The dbt Semantic Layer extends the portfolio of information managed in dbt. With the new semantic layer, you can find the information you need to make quality decisions. You can define metrics in your dbt project, and then data consumers can query those metrics in downstream tools.
+
+dbt is additionally working with partners to ensure that integrations with the dbt Semantic Layer meet organizations' needs. 
+
+Look [here](https://www.getdbt.com/blog/frontiers-of-the-dbt-semantic-layer/) for more information on the dbt Semantic Layer and integrations and use cases that the dbt Semantic Layer powers.
+
+___
+
+#Recap
+Now that we've seen the new features coming to dbt, I wanted to discuss some of my favorite sessions from the 100+ at Coalesce 2022.
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:55]: consider changing "wanted" to "want". 
consider changing "we've" to "we have".

File: posts/coalesce2022-a-recap.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+[Watch his talk here!](https://www.youtube.com/watch?v=gzr4CbeVY5s)
+[To install `QUERY;` go to the Apple App Store](https://apps.apple.com/us/app/query/id1636590940?mt=12)
+
+## Building Scalable Data Products leveraging User Stitching
+Tracking a user's journey can be rather tricky. Emiel Verkade from Snowplow discussed how Snowplow handles user tracking and user stitching in this talk. There are many ways people can enter into the domain of dbt. You can go directly to the docs site, or you are Googling the problem, which leads you to an answer you can try. You can also span multiple search sessions and multiple days to resolve a problem. Basic web analytics would look at each day's session separately. If you use user stitching, you can trace the pattern of use through a site and surmise if a visitor found a solution.
+
+At a high level, user stitching (or identity stitching) says that if two events belong to the same identifier, they belong to the same entity. In this way, multiple identifiers can be defined per customer. 
+
+How do you approach user stitching? Emiel recommends the following: 
+ -a set of properties to identify a user
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:72]:
```suggestion
- a set of properties to identify a user
```

File: posts/coalesce2022-a-recap.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+In this talk, Prukalpa Sankar, Co-founder of Atlan presents 20 scenarios in which data teams have used metadata to drive their work as well as how metadata and the dbt Semantic Layer can transform how a team works with metrics. 
+
+The chief question that Prukalpa and Atlan is trying to address is, `When I'm looking at data in a dashboard and am wondering if it is correct, is it possible to get the answers immediately?` 
+
+Let's get right into the 20 real-life use cases mentioned in the title:
+1. Streamline analyst 'service' requests
+> Problem: Constant questions and asks for analysts from business users. 
+> - What does metric X mean? How is it different from Metric Y? 
+> - Can someone confirm if this is the right way to join these tables? Here's the SQL statement
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:94]:
```suggestion
> - Can someone confirm if this is the right way to join these tables? Here's the SQL statement.
```

File: posts/coalesce2022-a-recap.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+> - I want to see MEtric X by geography instead of segment?
+> 
+> Solution: With metadata, you can actually go from requests to products. You can consider each of these requests as internal reusable products for their organization. Data, Metrics and Queries can all be products for your organization.
+
+2. Streamline Data/Analytics Engineer 'service' requests
+> Problem: Constant questions and asks for engineers from other data users
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:100]:
```suggestion
> Problem: Constant questions and asks for engineers from other data users.
```

File: posts/coalesce2022-a-recap.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+> Solution: You can extend data-as-a-product one step further. You can bring in context that a data engineer needs, such as Airflow SLAs (auto-updated), freshness from dbt tags, autoamted column level lineages, etc.
+
+3. Speed data team onboarding and ramp up
+> Problem: How do you speed up team onboarding and ramp-up?
+> Typically analysts spend 18 months on the job, and it takes them 6 months to onboard into an organization. 
+> 
+> Solution: By making assets (such as your data objects, metrics and queries) you can reduce this time astronomically.
+
+4. Speed up root-cause analysis
+> Problem: Finding the solution to `It doesn't seem like Metric X is right?` is far from easy
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:116]: Some "Problem" lines have punctuation some don't.  It's okay to leave off I suppose (like you do in the numbered headers), just make sure to do it consistently across the blog post.

File: posts/coalesce2022-a-recap.md [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+7. Active Change Alerting
+> Problem: How do you address changes to data in your platform?
+> 
+> Solution: If there is a change upstream, such as how Fivetran loads data, an alert propogates to the data team members as well as your business consumers.
+
+8. Data Platform Migrations
+> Problem: How do you migrate data?
+> As data engineers we migrate data all the time. How do you evaluate the complexity of a migration? How do you create deprecation and change notices? How do you make sure that nothing breaks on migration?
+> 
+> Solution: With metadata and a data lineage you can send notifications downstream to data team members and business consumers
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:142]: same here with on and off again punctuation.

File: posts/coalesce2022-a-recap.md [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+> Solution: You can integrate Atlan into your BI tool to show all of the metric context from the dbt Semantic layer right in the dashboard. So you have all of the data context to make decisions
+
+20. Enable data teams that can change the world
+> Problem: How do you unify and enable a diverse team to facilitate data to create change?
+> 
+> Solution: Embrace the diversity in a team. It is a data team's biggest strenght and weakness. Atlan's metadata is what can help a team gel with each other and do great things.
+
+[Watch Prukalpa's Full Talk here](https://www.youtube.com/watch?v=fxFxfKCZrhY&list=PL0QYlrC86xQlj9UDGiEwhXQuSjuSyPJHl&index=78)
+[For more information on all that Atlan can offer](https://atlan.com/)
+
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:224]: This recap of the 20 questions in the midst of the other recap summaries is a bit jarring.  I think it's probably fine to leave all 20 points, but consider hitting the top 3 or 5 instead.  Including all 20 points turns this blog post into a 10+ minute read.

File: posts/coalesce2022-a-recap.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+## Detecting Data Anomalies via an Inspection Layer
+In most of our current data ecosystems, we ingest data from various vendors, IoT devices, and more. Unfortunately, for every data source that is perfect, there is another (and we've all been there) where the data isn't what we were expecting. When the data has nulls, contains duplicates, or has a different schema than initially communicated, your data pipeline can stumble. You can handle this natively with dbt native methods and implement an inspection layer to ensure that 'bad' data is flagged and quarantined while the remainder is loaded uninterrupted.
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:226]: This is really cool!

File: posts/coalesce2022-a-recap.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+
+Neal Achord and John Rensberger from Red Pill Analytics walk us through their implementation of the inspection layer in this workshop. It was fascinating to see and implement this myself.
+
+You can follow along with the workshop by [watching their talk](https://www.youtube.com/watch?v=EKPi-T_nBio&t=80s), cloning the [repo](https://github.com/RedPillAnalytics/dbt_coalesce_workshop) on Github and referring to their [slides](https://docs.google.com/presentation/d/11Q9wwMfyz6xuxMXCPizFg4DKSY_zOIHPNOrsNI8oBn8/edit?usp=sharing).
+
+___
+I'm cognizant that all of the above is a lot. Thanks so much for sticking with me as I condensed a jam-packed week of content into a short little blog like this. I had a ton of fun and while I felt like I was drinking from a firehose at times, Coalesce was the best experience I've had at a conference to date. I thoroughly enjoyed getting to chat with data professionals and commiserating over all of the common data problems we face daily. All of their sessions and workshops where immensely thought-provoking and interesting. I hope I'll get to attend again sometime soon!
```
Comment (lward27) [inline posts/coalesce2022-a-recap.md:233]: Definitely a lot!!! Great write up. Thank you for bringing this recap to what seemed like a very impactful conference.  I absolutely love your writing style and think that your personality really showed through (a.k.a. not just an emotionless recap spewed on the page).

**Review by lward27 — APPROVED** [review]
Awesome Job! This legit made me want to check out dbt, to be sure.

File: posts/coalesce2022-a-recap.md (hunk 10 above)
Comment (pkrish123) [reply posts/coalesce2022-a-recap.md]: Thanks!

**Review by lanefu — COMMENTED** [review]
@pkrish123 Lots of great deets.    As and outsider it was hard for me to quickly understand what dbt is, and what they do.     

Could use a 1 or 2 liner kinda like what they drop in in a press release lead-in, but perhaps a little more descriptive. i just dug out this lol

`PHILADELPHIA, Feb. 24, 2022 /PRNewswire/ -- dbt Labs, the pioneer in analytics engineering, today announced that it has raised $222 million in Series D financing at a $4.2 billion valuation.`

**Pull request merged**